src/styles/normalize.scss

```diff
@@ -0,0 +1,17 @@
+html {
+  font-family: Roboto, Univa Nova, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+}
+
+body {
+  margin: 0;
+}
+
+h2 {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
```

Comment:
Try to remove it and than run tests, also try to move styles from html selector to body

Reply:
when i remove this,i have extra margin at the bottom,but i only assigned margin-top.I think the problem is in default margin,so i reseted this

Reply:
Can you explain the difference between assigning common styles to body and to html tags,please?